# HG changeset patch
# User Rik <rik@octave.org>
# Date 1370575188 25200
#      Thu Jun 06 20:19:48 2013 -0700
# Node ID 4da837c4902f5ed66a1f9db3cf05660ce8f700c4
# Parent  34728454d8a68b706d4d2396f4b8823d173f5a67
NEWS: alphabetize functions added in 3.8.0.

* NEWS: alphabetize functions added in 3.8.0.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -171,21 +171,21 @@ Summary of important user-visible change
     are now available (provided that Octave is compiled with support for
     Java enabled):
 
       helpdlg    listdlg   questdlg
       inputdlg   msgbox    warndlg
 
  ** Other new functions added in 3.8.0:
 
-      betaincinv   ellipj    findfigs     polyeig       tetramesh
-      cmpermute    ellipke   fminsearch   rgbplot       waterfall
-      cmunique     erfcinv   importdata   shrinkfaces   gallery
-      colorcube    erfi      iscolormap   splinefit
-      dawson       expint    lines        strjoin
+      betaincinv   ellipj    findfigs     lines          strjoin
+      cmpermute    ellipke   fminsearch   polyeig        tetramesh
+      cmunique     erfcinv   gallery      rgbplot        waterfall
+      colorcube    erfi      importdata   shrinkfaces
+      dawson       expint    iscolormap   splinefit  
 
  ** Deprecated functions.
 
     The following functions were deprecated in Octave 3.4 and have been
     removed from Octave 3.8.
                                            
       autocor    dispatch              is_global    strerror
       autocov    fstat                 krylovb      values  
